server/src/main/java/io/druid/guice/AWSModule.java

```diff
@@ -47,8 +50,17 @@ public AWSCredentialsProvider getAWSCredentialsProvider(final AWSCredentialsConf
 
   @Provides
   @LazySingleton
-  public AmazonEC2 getEc2Client(AWSCredentialsProvider credentials)
+  public AmazonEC2 getEc2Client(
+      AWSCredentialsProvider credentials,
+      ResourceManagementSchedulerConfig resourceManagementSchedulerConfig)
   {
-    return new AmazonEC2Client(credentials);
+    final AmazonEC2Client amazonEC2Client = new AmazonEC2Client(credentials);
+    final String region = resourceManagementSchedulerConfig.getRegion();
+    if (region != null) {
+      try {
+        amazonEC2Client.setRegion(Region.getRegion(Regions.fromName(region)));
```

Comment:
the problem with changing region globally is that this will also change the region for other AWS services (such as S3), not just autoscaling, and could cause some unintended behavior.
I believe we should have separate ec2 client providers for auto-scaling and s3, and be able to configure region separately.